RHEL/7/input/xccdf/system/accounts/physical.xml

```diff
@@ -129,6 +129,56 @@ the grub2 superuser account and password, please refer to
 <tested by="DS" on="20121026"/>
 </Rule>
 
+<Rule id="bootloader_uefi_password" severity="medium">
+<title>Set the UEFI Boot Loader Password</title>
+<description>The UEFI grub2 boot loader should have a superuser account and password
+protection enabled to protect boot-time settings.
+<br /><br />
+To do so, select a superuser account and password and add them into the
+appropriate grub2 configuration file(s) under <tt>/etc/grub.d</tt>.
```

Comment:
Please explicitly mention ```/etc/grub.d/01_users file``` file here (to be consistent with the official documentation).